app/controllers/concerns/foreman/controller/parameters/nic_base.rb

```diff
@@ -24,6 +23,7 @@ def add_nic_base_params_filter(filter)
         :username,
         :virtual,
         {:attached_devices => []},
+        {:compute_attributes => [:bridge, :from_profile, :model, :network, :type, :name]},
```

Comment:
`compute_attributes` is generally passed through without filtering as the attributes are different for every UI, including plugins, e.g. in the `NicInterface` concern using `keep_param`.

That keep_param helper only works for the first level of hashes at the moment though, so it might need extending somehow to work in `Host` for nested interfaces + compute_attributes.